Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.py

```diff
@@ -305,27 +409,92 @@ def list_findings(client: boto3.client, args: dict):
                           outputs_key_field='')
 
 
+def get_pagination_args(args: dict):
+    """
+    Gets and validates pagination arguments.
+    :param args: The command arguments (page, page_size or limit)
+    :return: limit, page_size, page after validation and convert
+    """
+
+    # Automatic Pagination
+    limit = arg_to_number(args.get('limit', MAX_RESULTS_RESPONSE))
+
+    # Manual Pagination
+    page = arg_to_number(args['page']) if args.get('page') else None
```

Comment:
```suggestion
    page = arg_to_number(args.get('page'))
```
if `arg_to_number` gets None it will return None